src/lib.rs

```diff
@@ -8,55 +8,3 @@ pub mod properties;
 pub mod tile;
 pub mod tileset;
 mod util;
-
-use base64;
-
-use error::*;
-use image::*;
-use layers::*;
-use map::*;
-use std::collections::HashMap;
-use std::fmt;
-use std::fs::File;
-use std::io::Read;
-use std::path::{Path, PathBuf};
-use std::str::FromStr;
-use tile::*;
-use tileset::*;
-use util::*;
-use xml::attribute::OwnedAttribute;
-use xml::reader::XmlEvent;
-use xml::reader::{Error as XmlError, EventReader};
-
-// TODO move these
-/// Parse a buffer hopefully containing the contents of a Tiled file and try to
-/// parse it. This augments `parse` with a file location: some engines
-/// (e.g. Amethyst) simply hand over a byte stream (and file location) for parsing,
-/// in which case this function may be required.
-pub fn parse_with_path<R: Read>(reader: R, path: &Path) -> Result<Map, TiledError> {
-    parse_impl(reader, Some(path))
-}
-
-/// Parse a file hopefully containing a Tiled map and try to parse it.  If the
-/// file has an external tileset, the tileset file will be loaded using a path
-/// relative to the map file's path.
-pub fn parse_file(path: &Path) -> Result<Map, TiledError> {
-    let file = File::open(path)
-        .map_err(|_| TiledError::Other(format!("Map file not found: {:?}", path)))?;
-    parse_impl(file, Some(path))
-}
-
-/// Parse a buffer hopefully containing the contents of a Tiled file and try to
-/// parse it.
-pub fn parse<R: Read>(reader: R) -> Result<Map, TiledError> {
```

Comment:
Move of `parse` needs to be reflected in the `README.md`.

Reply:
It's in the changelog. I could add a notice for old users pointing to the CHANGELOG file.

Reply:
I just meant, the `README.md` needs to be updated, because it contains an example that still uses the old API.